### PR TITLE
[REF] Regen following #25211

### PR DESCRIPTION
**Pull request by seamuslee001**
@demeritcowboy

### CODE REVIEW

File: install/langs.php [hunk 1]
```diff
@@ -6,6 +6,7 @@
   'ca_ES' => 'Catalan; Valencian',
   'zh_CN' => 'Chinese (China)',
   'zh_TW' => 'Chinese (Taiwan)',
+  'hr_HR' => 'Croatian',
```
Comment (demeritcowboy) [inline install/langs.php:9]: Right this is from 5.58 that never got regen'd I think. ✅

**Comment by demeritcowboy** [comment]
Something's not right. Checking...

**Comment by demeritcowboy** [comment]
See comment on other PR: https://github.com/civicrm/civicrm-core/pull/25211/files#r1086067714

**Comment by seamuslee001** [comment]
@demeritcowboy I think I have fixed those issues in the new commit

**Comment by demeritcowboy** [comment]
Woo green!

**Pull request merged**